client/data/deposits/test/reducer.js

```diff
@@ -130,40 +125,6 @@ describe( 'Deposits reducer tests', () => {
 		expect( reduced ).toStrictEqual( after );
 	} );
 
-	test( 'Deposits overview is reduced correctly', () => {
-		const reduced = reducer(
-			undefined, // Default state.
-			{
-				type: types.SET_DEPOSITS_OVERVIEW,
-				data: mockOverview,
-			}
-		);
-
-		expect( reduced ).toStrictEqual( {
-			...mockEmptyState,
-			overview: {
-				data: mockOverview,
-			},
-		} );
-	} );
-
-	test( 'Deposits overview error is reduced correctly', () => {
-		const reduced = reducer(
-			undefined, // Default state.
-			{
-				type: types.SET_ERROR_FOR_DEPOSITS_OVERVIEW,
-				error: { code: 'error' },
-			}
-		);
-
-		expect( reduced ).toStrictEqual( {
-			...mockEmptyState,
-			overview: {
-				error: { code: 'error' },
-			},
-		} );
-	} );
-
```

Comment:
I was not aware we had tests for reducers! I wonder how many issues these have caught.